src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -329,6 +329,11 @@ configurationRegistry.registerConfiguration({
 			default: 1,
 			markdownDescription: nls.localize('search.searchEditor.defaultNumberOfContextLines', "The default number of surrounding context lines to use when creating new Search Editors. If using `#search.searchEditor.reusePriorSearchConfiguration#`, this can be set to `null` (empty) to use the prior Search Editor's configuration.")
 		},
+		'search.searchEditor.focusResults': {
+			type: 'boolean',
+			default: false,
+			markdownDescription: nls.localize('search.searchEditor.focusResults', "Focus the Search Editor results instead of the Search Editor input.")
+		},
```

Comment:
I got https://github.com/microsoft/vscode/pull/214289 merged in, so this PR should be pretty much ready to be merged in. One final thing: maybe we should make this clearer that this is where it focuses on search. So maybe calling it `search.searchEditor.focusResultsOnSearch` and mentioning that the setting applies when you trigger a search.